sopel/db.py

```diff
@@ -4,9 +4,14 @@
 import json
 import os.path
 import sys
-import sqlite3
 
-from sopel.tools import Identifier
+from sopel import tools
```

Comment:
Changing this import kind of cluttered the diff unnecessarily, since nothing else from `sopel.tools` is used in this file.

`Identifier()` as used throughout this module is basically type-casting. None of Python's built-in casting operations contain a dot (think `str()`, `int()`, `list()`, etc.), and I don't like putting one in ours.

Reply:
This is legacy from the old db module. If you have a better solution for it, I am happy to implement.

Reply:
What's legacy? Are we talking about different changes, maybe?

Reply:
I am specifically referring to the Identifier stuff (which has been in the codebase for like 5 years). I've updated the import to be less gross.

Reply:
We covered this one on IRC, and figured out the disconnect. :heart: